setup.py

```diff
@@ -1,128 +0,0 @@
-#!/usr/bin/env python3
-# SPDX-License-Identifier: BSD-3-Clause
-
-from pathlib    import Path
-
-from setuptools import find_packages, setup
-
-REPO_ROOT   = Path(__file__).parent
-README_FILE = (REPO_ROOT / 'README.md')
-
-def scm_version():
-	def local_scheme(version):
-		if version.tag and not version.distance:
-			return version.format_with('')
-		else:
-			return version.format_choice('+{node}', '+{node}.dirty')
-	return {
-		'relative_to'   : __file__,
-		'version_scheme': 'guess-next-dev',
-		'local_scheme'  : local_scheme
-	}
-
-def doc_version():
-	try:
-		from setuptools_scm.git import parse as parse_git
-	except ImportError:
-		return ''
-
-	git = parse_git('.')
-	if not git:
-		return ''
-	elif git.exact:
-		return git.format_with('v{tag}')
-	else:
-		return 'latest'
-
-setup(
-	name             = 'torii-usb',
-	use_scm_version  = scm_version(),
-	author           = 'Katherine Temkin',
-	author_email     = 'k@ktemkin.com',
-	maintainer       = ', '.join([
-		'Aki Van Ness',
-		'Rachel Mant',
-	]),
-	maintainer_email = ', '.join([
-		'aki@lethalbit.net',
-		'git@dragonmux.network',
-	]),
-	license          = 'BSD-3-Clause',
-	description      = 'Torii HDL framework for FPGA-based USB solutions',
-	python_requires  = '~=3.10',
-	zip_safe         = True,
-	url              = 'https://torii-usb.shmdn.link/',
-
-	long_description = README_FILE.read_text(),
-	long_description_content_type = 'text/markdown',
-
-	setup_requires = [
-		'wheel',
-		'setuptools',
-		'setuptools_scm',
-	],
```

Comment:
These bounds for the project appear to be missing in the new pyproject.toml? If that's okay then that's fine, but it doesn't seem right..